SingularityUI/app/actions/api/history.es6

```diff
@@ -3,8 +3,9 @@ import Utils from '../../utils';
 
 export const FetchTaskHistory = buildApiAction(
   'FETCH_TASK_HISTORY',
-  (taskId) => ({
-    url: `/history/task/${taskId}`
+  (taskId, isMainApiCall) => ({
+    url: `/history/task/${taskId}`,
+    isMainApiCall
```

Comment:
Naming things isn't my strong suit. Maybe `pageCantRenderIfThisFails`? I don't really like that name either though.